articles/article11_rag.md

```
@@ -0,0 +1,237 @@
+## 前提構成
+
+| 役割 | 技術 |
+|------|------|
+| 情報源 | Notion |
+| ベクトルDB | Supabase (Postgres + pgvector) |
+| アプリケーション | Next.js (App Router) |
+| AI | OpenAI (埋め込み + チャット) |
+
+リポジトリはこちらです 👇
+
+https://github.com/KazusaNakagawa/test-rag-prj
```

markdownlint-cli2 (0.20.0)

[warning] Bare URL used

(MD034, no-bare-urls)